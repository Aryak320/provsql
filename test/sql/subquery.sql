\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

CREATE TABLE subquery_result AS
SELECT name AS name1, name2
FROM (
  SELECT p1.*, p2.name AS name2
  FROM personnel p1, personnel p2
  WHERE p1.city=p2.city AND p1.name<>p2.name
) t;

SELECT remove_provenance('subquery_result');

SELECT * FROM subquery_result
ORDER BY name1, name2;
