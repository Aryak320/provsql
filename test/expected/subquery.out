\set ECHO none
remove_provenance

(1 row)
name1|name2
Dave|Magdalen
Dave|Nancy
Ellen|Susan
John|Paul
Magdalen|Dave
Magdalen|Nancy
Nancy|Dave
Nancy|Magdalen
Paul|John
Susan|Ellen
(10 rows)
